src/com/projectkorra/projectkorra/command/HelpCommand.java

```diff
@@ -214,4 +228,8 @@ protected List<String> getTabCompletion(final CommandSender sender, final List<S
 		list.addAll(abils);
 		return list;
 	}
+	
+	private final String color(String toColor) {
```

Comment:
I believe we got a method for this in ChatUtil that also covers more stuff for hex colors. Not sure if it's applicable or not

Reply:
Will take a look

Reply:
You were in fact correct, went ahead and replaced my method with that